extensions/raindrop-io/src/components/BookmarkItem.tsx

```diff
@@ -96,7 +96,29 @@ export default function BookmarkItem(props: { bookmark: Bookmark; revalidate: ()
   function accessories() {
     const accessories = [];
 
-    bookmark.tags.forEach((tag) => accessories.push({ tag: `#${tag}` }));
+    switch (preferences.displayDate) {
+      case "lastUpdated":
+        accessories.push({ date: lastUpdatedDate, tooltip: lastUpdatedDate.toLocaleString() });
+        break;
+      case "created":
+        accessories.push({ date: createdDate, tooltip: createdDate.toLocaleString() });
+        break;
+    }
+
+    if (preferences.tagsDisplay !== "none") {
+      if (preferences.tagsDisplay === "all") {
+        bookmark.tags.forEach((tag) => accessories.push({ tag: `#${tag}` }));
+      } else {
+        const tagsNumber = bookmark.tags.length - 1;
+        const tagsDisplayNumber = parseInt(preferences.tagsDisplay, 10);
+        if (tagsNumber === tagsDisplayNumber) {
+          bookmark.tags.forEach((tag) => accessories.push({ tag: `#${tag}` }));
+        } else {
+          bookmark.tags.slice(0, tagsDisplayNumber).forEach((tag) => accessories.push({ tag: `#${tag}` }));
+          accessories.push({ tag: `+${tagsNumber}`, tooltip: bookmark.tags.join("\n") });
+        }
```

Comment:
I think you can simplify a little this code by doing something like this:

```suggestion
        const displayedTags = bookmark.tags.slice(0, tagsDisplayNumber).map(tag => `#${tag}`);
        accessories.push(...displayedTags.map(tag => ({ tag })));
        
        if (tagsNumber > tagsDisplayNumber) {
            accessories.push({ tag: `+${tagsNumber}`, tooltip: bookmark.tags.join("\n") });
        }
```

Reply:
Thank you for suggestion. Commited refactor :)